lib/widget/download/download_info_dialog.dart

```diff
@@ -67,8 +67,6 @@ class _DownloadInfoDialogState extends State<DownloadInfoDialog> {
                                   widget.downloadItem.fileType),
                               width: 70,
                               height: 70,
-                              color: FileUtil.resolveFileTypeIconColor(
```

Comment:
Please roll this change back. Removing the color field would result in black icons which doesn't look good.

Reply:
i fix this issue